projects/product-configuration/src/test.ts

```diff
@@ -0,0 +1,30 @@
+// This file is required by karma.conf.js and loads recursively all the .spec and framework files
+
+import 'zone.js/dist/zone';
```

Comment:
I believe you are using some i18n stuff in your UI layer, right?
Let's add the `import '@angular/localize/init';`.